src/MessengerCustomerChat.js

```diff
@@ -95,12 +95,12 @@ export default class MessengerCustomerChat extends Component {
 
   removeFacebookSDK() {
     const fbjssdk = document.getElementById('facebook-jssdk');
-    if (fbjssdk) {
-      document.body.removeChild(fbjssdk);
+    if (fbjssdk.parentNode) {
+      fbjssdk.parentNode.removeChild(fbjssdk);
     }
     const fbroot = document.getElementById('fb-root');
-    if (fbroot) {
-      document.body.removeChild(fbroot);
+    if (fbroot.parentNode) {
```

Comment:
In some case, fbroot can be null at the first load, I suggest to check null for fbroot before getting parentNode